libraries/standard/http/test/test-HTTPClient_AddHeader.c

```diff
@@ -1,9 +1,54 @@
 #include "common.h"
```

Comment:
You will have to make new files for the other functions. (you probably already know this)